.github/workflows/test-all.yml

```diff
@@ -185,7 +185,7 @@ jobs:
         with:
           jsengine: ${{ matrix.jsengine }}
           architecture: ${{ matrix.architecture }}
-          run-unit-tests: "false"
+          run-unit-tests: true
```

Comment:
the problem with this is that we will run unit tests on ALL the matrix configuration. This is a bit of an overkill and it will increase the costs of CI sensibly.

I believe it would be better to copy this job in the workflow and to run unit tests with the Hermes, New Architecture, Static Library configuration only. This should be enough.

Reply:
> copy this job in the workflow

Instead of making a copy, I'd suggest adding this as a parameter to the matrix and [use the `include` feature](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations) to run it only on the configurations you mentioned.

Would you be okay with keeping things simple for now and ☝️ to be a separate PR?

Reply:
can you move this to matrix in this pr?
This is not pretty complex, and adding a param to the matrix should not add a lot of noise.

Reply:
> Hermes, New Architecture, Static Library configuration only

For both `Debug` and `Release` or just one of these?